storybookjs/storybook
# Feature Wix example in docs

Issue: N/A

## What I did

Add wix styleguide

## How to test

```
yarn bootstrap:docs
yarn docs:dev
open http://localhost:8000/examples/
```

Is this testable with jest or storyshots?

Does this need a new example in the kitchen sink apps?

Does this need an update to the documentation?

If your answer is yes to any of these, please make sure to include it in your PR.
